utils/tests_fetcher.py

```diff
@@ -1244,7 +1244,7 @@ def parse_commit_message(commit_message: str) -> Dict[str, bool]:
                     args.output_file,
                     diff_with_last_commit=diff_with_last_commit,
                     json_output_file=args.json_output_file,
-                    filter_models=(not commit_flags["no_filter"] or is_main_branch),
```

Comment:
the previous one here was wrong: this actually filter whenever it is on `main` branch.